lib/routes/account.js

```diff
@@ -1646,11 +1668,46 @@ module.exports = function (
             isVerified = isVerified && sessionVerified
           }
 
-          return {
-            email: sessionToken.email,
-            verified: isVerified,
-            sessionVerified: sessionVerified,
-            emailVerified: emailVerified
+          if (! isVerified) {
+            return bounces.check(sessionToken.email)
```

Comment:
Yeah, this might pose a problem. We alert when we go above 1000 read IOPS and we see an daily swing of 250-750 read iops.